test/src/cli/flutter_cli_test.dart

```diff
@@ -459,7 +467,8 @@ void main() {
                   'test/counter/cubit/counter_cubit_test.dart 16:7     main.<fn>.<fn>\n',
               '\x1B[2K\rCounterCubit initial state is 0 /my_app/test/counter/cubit/counter_cubit_test.dart (FAILED)',
               '\x1B[2K\rFailing Tests:\n'
-                  '\x1B[2K\r - [FAILED] test/counter/cubit/counter_cubit_test.dart:16:7\n'
+                  '\x1B[2K\r - /my_app/test/counter/cubit/counter_cubit_test.dart \n'
```

Comment:
Does this `/` mean that it will always show the absolute path, I wonder if we can get the relative path based on the current directory. That would make the test output cleaner while keeping the info still useful

Reply:
Yes, in the final report, we show always the absolute path and failing tests by each file. My purpose is that the final report keep the same format in the log before the summary (in the log, full path is used).   Paths are clickable from terminal inde IDE  to open it. However, I agree that is information is less useful in case of log on CI.  

I looked in the code how I can get only relative path. I didn't find a reliable approach because we can have also recursive excution via "--recursive" flag.

Reply:
We can just make it relative to the `Directory.current`, which will work for the `--recursive` flag as that runs from the current directory

Reply:
Done using path.relative. Fixtures changed to  be dynamic according to command working directory..
@renancaraujo  After rebase, I lost your approval ^^

Reply:
@wolfenrain @renancaraujo are there any other points  to deal with?